app/javascript/components/recipes/new/selected-ingredients.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="flex w-80 flex-col">
+    <span class="flex content-center justify-center">Ingredientes seleccionados</span>
+    <div 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Trailing spaces not allowed                            no-trailing-spaces